Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -1265,30 +1269,29 @@ static ErrorOr<void> read_start_of_frame(JPEGStream& stream, JPEGLoadingContext&
         component.index = i;
 
         u8 subsample_factors = TRY(stream.read_u8());
-        component.hsample_factor = subsample_factors >> 4;
-        component.vsample_factor = subsample_factors & 0x0F;
+        component.sampling_factors = SamplingFactors(subsample_factors >> 4, subsample_factors & 0x0F);
```

Comment:
https://dev.azure.com/SerenityOS/SerenityOS/_build/results?buildId=42764&view=logs&jobId=2351e412-c793-598f-27f8-72c4f2d55ba1&j=2351e412-c793-598f-27f8-72c4f2d55ba1&t=9c50333a-ada2-5596-d44b-d8dcee270ad2

Probably have to use braced init here (?)

Reply:
Brace init can't work as both value are `int` and not `u8`

Reply:
Just have to add a cast (?)